[AUTOGENERATED] [release/2.7] [SWDEV-523736] Skip&Fix some testcases for archs without SDPA or Navi4x

okakarpa opened the pull request:
Cherry-pick of https://github.com/ROCm/pytorch/pull/2213 
Need to resolve conflicts

k-artem committed:
Cherry-picked commit with merge conflict

pruthvistony commented:
@k-artem ,
Fix the merge conflicts.

k-artem commented:
> @k-artem , Fix the merge conflicts.

done

k-artem committed:
[release/2.7] Fix merge conflicts

iupaikov-amd commented:
@k-artem Hello! Do you have and ETA on this PR? Got some jiras about attention tests for gfx1101, they will be all fixed by merging this. Also any plans on release/2.8 cherry-pick?

Regards, Iurii.

k-artem committed:
Merge branch 'release/2.7' into autogenerated/release/2.7_cherry-pick_pr-2213

k-artem committed:
Update test_distributed_checkpoint.py

Sync with upstream https://github.com/pytorch/pytorch/commit/bfc27cf468660b50758defdc86c5d19df8750c2e
